index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const app = express();
+
+const PORT = 3000;
+const HOST = 'localhost';
+
+app.use(express.static(__dirname));
+
+app.get('/info', (req, res, next) => {
+  res.send('This is a proxy service which proxies to the 4 services present in the Team 2 FEC project.');
+});
+
+app.listen(PORT, HOST, () => {
+  console.log(`Starting Proxy at ${HOST}:${PORT}`);
```

Comment:
I knew about using the port like this but never thought of doing it for the host. Very cool!